src/provider/application.js

```diff
@@ -23,14 +23,17 @@ class Application extends EventEmitter {
     this.secret = secret;
     this.onReady = onReady;
     this.targetSelectors = targetSelectors;
-    this.resizeConfig = null;
+    this.resizeConfig = {};
```

Comment:
Actually thinking about this. We should probably leave this null. Then in "imageRequestResize" if resize config isn't set it should do nothing. This means the SDK hasn't done initial resize and will spare computation to redundantly do it for the image and then again for SDK as a whole.

Reply:
alternative is to make `requestResize` return out iif config isn't set